bin/maf/metadata_dir

```diff
@@ -53,134 +23,157 @@ if __name__ == "__main__":
     """
     parser = argparse.ArgumentParser()
     parser.add_argument("--db", type=str, default=None)
+    parser.add_argument(
+        "--nside",
+        type=float,
+        default=64,
+        help="nside to use for the healpix slicer and subsetslicer for metrics",
+    )
     args = parser.parse_args()
 
     # If runNames not given, scan for opsim databases in current directory and use those
     # Note that 'runNames' can be full path to directories
 
     if args.db is None:
         # Just look for any .db files in this directory
-        dbFiles = glob.glob('.db')
+        dbFiles = glob.glob("*.db")
         # But remove trackingDb and resultsDb if they're there
         try:
-            dbFiles.remove('trackingDb_sqlite.db')
+            dbFiles.remove("trackingDb_sqlite.db")
         except ValueError:
             pass
         try:
-            dbFiles.remove('resultsDb_sqlite.db')
+            dbFiles.remove("resultsDb_sqlite.db")
         except ValueError:
             pass
     elif isinstance(args.db, str):
         dbFiles = [args.db]
     else:
         dbFiles = args.db
 
-    sim_names = [os.path.basename(name).replace('.db', '') for name in dbFiles]
+    sim_names = [os.path.basename(name).replace(".db", "") for name in dbFiles]
 
-    trackingDb = db.TrackingDb(database=None)
-    mafDate, mafVersion = mafUtils.getDateVersion()
-    mafVersion = mafVersion['__version__']
+    trackingDb = TrackingDb(database=None)
+    mafDate, mafVersion = getDateVersion()
+    mafVersion = mafVersion["__version__"]
 
     for filename, opsim in zip(dbFiles, sim_names):
-        # label visits
-        newdb_file = mafUtils.labelVisits(filename)
+        # Connect to the database
+        opsdb = filename
+        colmap = batches.ColMapDict()
+
         # Set and create if needed the output directory
         # I guess 'meta' really means to be more like 'conditions' of visits here.
         outDir = opsim + "_meta"
         if os.path.isdir(outDir):
             shutil.rmtree(outDir)
-        # Connect to the opsim database
-        opsdb = db.OpsimDatabase(newdb_file)
-        colmap = batches.ColMapDict()
 
-        # Set up the bundle dicts
-        # Set up WFD sql constraint, if possible.
-        propids, proptags, sqls, metadata = setSQL(opsdb)
-        if 'WFD' in sqls:
-            tags = ['All', 'WFD']
-        else:
-            tags = ['All']
+        # Find the 'wfd' footprint
+        m = CountExplimMetric(col="observationStartMJD")
+        allsky_slicer = HealpixSlicer(nside=args.nside)
+        constraint = 'note not like "%DD%"'
+        bundle = MetricBundle(m, allsky_slicer, constraint, runName=opsim)
+        g = MetricBundleGroup({f"{opsim} footprint": bundle}, opsdb, outDir=outDir)
+        g.runAll()
+        wfd_footprint = bundle.metricValues.filled(0)
+        wfd_footprint = np.where(wfd_footprint > 750, 1, 0)
```

Comment:
magic number 750 should probably be an arg so it's documented as well.

Reply:
True!